sdk/qrcode/android/java/com/google/cardboard/sdk/qrcode/CardboardParamsUtils.java

```diff
@@ -20,6 +20,7 @@
 import android.os.Build.VERSION;
 import android.os.Build.VERSION_CODES;
 import android.os.Environment;
+import android.support.annotation.Nullable;
```

Comment:
Consider checking whether any of these added dependencies can now be removed from `QrCodeContentProcessor.java`.